docs/Specifications/Workgroup Creation & Management/Interfaces and schemas for the main logical input objects and resolvers.md

```diff
@@ -0,0 +1,233 @@
+# Workgroup creation and management - interfaces and schemas for the main logical input objects and resolvers
+
+# Overview
+
+The goal of this specification is to outline the current state of the main logical input objects that are needed for the creation and management of a baseline workgroup. It also defines the ways these inputs are to be resolved and highlights the extensibility of the resolving approach.
+
+The current version of this document is hugely influenced by the Radish34 demo use-case and we expect it to be gradually becoming more generic with the addition of new use-cases and abstracting Radish34 into Baseline.
+
+## Logical Inputs
+
+The logical inputs in a workgroup creation and management are input parameters that have to be provided to be provided from the workgroup management body in order for certain actions to be done. The concrete inputs required by the various actions are yet to be defined in another specification. This document will highlight all the main logical inputs - Keystore, organisation config, zero-knowledge circuit verifier key, and contract artefacts.
```

Comment:
"to be provided to be provided" typo

"...certain actions to be done." -> "...certain actions to be taken." (reads slightly better and easy to fix when the typo is corrected ;) )